modules/sdk-coin-avaxc/test/unit/avaxc.ts

```diff
@@ -955,5 +957,183 @@ describe('Avalanche C-Chain', function () {
+          tokenContractAddress,
+          gasPrice: gasPrice.toString(),
+        };
+        const recoveryTxn = await tavaxCoin.recover(params);
+
+        recoveryTxn.should.not.be.undefined();
+        recoveryTxn.should.have.property('txHex');
+        const txBuilder = tavaxCoin.getTransactionBuilder() as TransactionBuilder;
+        txBuilder.from(recoveryTxn.txHex);
+        const tx = await txBuilder.build();
+        const recoveryAmount = endpointResponses.addressTokenBalanceResponse.result;
+        tx.toBroadcastFormat().should.not.be.empty();
+        tx.inputs.should.not.be.empty();
+        tx.inputs[0].address.should.equal(recoveryUsers.coldWalletRecoveryUser.walletContractAddress);
+        tx.inputs[0].value.should.equal(recoveryAmount);
+        tx.inputs[0].should.have.property('coin');
+        tx.inputs[0].coin?.should.equal(tokenName);
+        tx.outputs.should.not.be.empty();
+        tx.outputs[0].address.should.equal(destAddr);
+        tx.outputs[0].value.should.equal(endpointResponses.addressTokenBalanceResponse.result);
+        tx.outputs[0].should.have.property('coin');
+        tx.outputs[0].coin?.should.equal(tokenName);
+
+        recoveryTxn.should.have.property('userKey');
+        recoveryTxn.userKey.should.equal(recoveryUsers.coldWalletRecoveryUser.userKey);
+        recoveryTxn.should.have.property('backupKey');
+        recoveryTxn.backupKey.should.equal(recoveryUsers.coldWalletRecoveryUser.backupKey);
+
+        recoveryTxn.should.have.property('coin');
+        recoveryTxn.coin.should.equal('tavaxc');
+
+        recoveryTxn.should.have.property('gasPrice');
+        recoveryTxn.gasPrice.should.equal(gasPrice.toString());
+        recoveryTxn.should.have.property('gasLimit');
+        recoveryTxn.gasLimit.should.equal('500000');
+
+        recoveryTxn.should.have.property('recipients');
+        recoveryTxn.recipients.length.should.equal(1);
+        recoveryTxn.recipients[0].address.should.equal(destAddr);
+        recoveryTxn.recipients[0].amount.should.equal(recoveryAmount);
+
+        recoveryTxn.should.have.property('walletContractAddress');
+        recoveryTxn.walletContractAddress.should.equal(recoveryUsers.coldWalletRecoveryUser.walletContractAddress);
+
+        recoveryTxn.should.have.property('amount');
+        recoveryTxn.amount.should.equal(recoveryAmount);
+
+        recoveryTxn.should.have.property('recipient');
+        recoveryTxn.recipient.address.should.equal(destAddr);
+        recoveryTxn.recipient.amount.should.equal(recoveryAmount);
+
+        recoveryTxn.should.have.property('tokenContractAddress');
+        recoveryTxn.tokenContractAddress.should.equal(tokenContractAddress);
+
+        recoveryTxn.should.have.property('backupKeyNonce');
+        recoveryTxn.should.have.property('expireTime');
+        recoveryTxn.should.have.property('contractSequenceId');
+        recoveryTxn.should.have.property('nextContractSequenceId');
```

Comment:
An array of property keys can be created that can be looped over and all these `recoveryTxn.should.have.property` assertions can be done in 1 single loop, leading to less duplication.